app/views/layouts/application.html.erb

```diff
@@ -1,6 +1,6 @@
 <!DOCTYPE html>
 <html> <head>
-<title><%= yield :title %></title>
+<title><%= full_title(yield :title) %></title>
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
```

Comment:
csp là gì hả em?

Reply:
Content Security Policy (CSP) là 1 cách hữu hiệu để giảm thiểu hoặc loại bỏ hoàn toàn các lỗ hổng Cross Site Scripting (XSS). Với CSP, chúng ta có thể chặn inline script và các script từ những nguồn không đáng tin cậy. Chúng ta định nghĩa policy thông qua 1 HTTP header chứa các rule dành cho tất cả các loại tài nguyên.
Mặt khác, điều đó cũng có nghĩa là chúng ta phải chuyển các đoạn inline script ra các file riêng biệt. Dù sao thì đây cũng là 1 điều nên làm và nó sẽ cho phép tái sử dụng 1 lượng code lớn hơn.